extensions/wikia/ArticleVideo/ArticleVideo.i18n.php

```diff
@@ -4,8 +4,10 @@
 
 $messages['en'] = [
 	'articlevideo-watch' => 'watch • $1',
+	'articlevideo-related-video' => 'Related video',
```

Comment:
To make the order customizable by translations. But that's true if we want to split that in the future we should have these separated.